Feat/workshops modal confirmation

PiotrDrechsler opened the pull request.

PiotrDrechsler committed:
basic structure and styles

add structure and styles

PiotrDrechsler committed:
translation pl

add polish version

PiotrDrechsler committed:
translation eng

add eng version

PiotrDrechsler committed:
anchors

add anchors

Okazaki92 commented on src/components/WorkshopsForm/WorkshopFormModalContent.jsx:
```diff
@@ -19,7 +20,8 @@ const ModalContent = () => {
   ]
   const radioOptions = ['Warsztat 1', 'Warsztat 2', 'Warsztat 3', 'Warsztat 4']
-  const {openModal, isModalVisible, setModalContent} = useModal()
+  const {openModal, isModalVisible, setModalContent, setIsModalVisible} =
```
setIsModalVisible do wywalania zamiast tego closeModal

PiotrDrechsler replied:
O faktycznie :D Teraz wszystko jest ok. Poprawione i śmiega. Merguje!

Okazaki92 commented on src/components/WorkshopsForm/WorkshopFormModalContent.jsx:
```diff
@@ -47,18 +49,32 @@ const ModalContent = () => {
+            content={t('workshopsForm.buttonModal1')}
+            onClick={() => {
+              scrollToAnchor('program')
+              setIsModalVisible(false)
```
tutaj zamiast tego setModalVisible musisz dać closeModal() bo w tej chwili jak klikniesz na buttona jednego albo drugiego to nie działa scroll strony

Okazaki92 commented on src/components/WorkshopsForm/WorkshopFormModalContent.jsx:
```diff
@@ -47,18 +49,32 @@ const ModalContent = () => {
+            content={t('workshopsForm.buttonModal2')}
+            onClick={() => {
+              scrollToAnchor('newsletter')
+              setIsModalVisible(false)
```
j.w

Okazaki92 reviewed (changes requested):
mały rzecz do poprawy. Reszta Ok :D

PiotrDrechsler committed:
fix/no scroll bug

fix bug with no scroll on main page after clicking an button on modal

Vincent's notes
